<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1084</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Janela Principal</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Teste</string>
    </property>
   </widget>
   <widget class="QLabel" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>514</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time Amarelo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_atacando">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Atacando --&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1084</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionNovo"/>
    <addaction name="actionCarregar"/>
    <addaction name="actionSalvar"/>
    <addaction name="actionSalvar_como"/>
    <addaction name="separator"/>
    <addaction name="actionFechar"/>
   </widget>
   <widget class="QMenu" name="menuCalibrar">
    <property name="title">
     <string>Calibrar</string>
    </property>
    <addaction name="actionCores"/>
    <addaction name="actionDistancias"/>
    <addaction name="actionBordas_da_arena"/>
    <addaction name="actionTIme_Amarelo"/>
    <addaction name="actionTime_Azul"/>
    <addaction name="actionFlip"/>
    <addaction name="actionTrocar_Atacante"/>
   </widget>
   <widget class="QMenu" name="menuAjuda">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionSobre"/>
   </widget>
   <widget class="QMenu" name="menuComandos">
    <property name="title">
     <string>Comandos</string>
    </property>
    <addaction name="actionIniciar"/>
    <addaction name="actionParar"/>
    <addaction name="actionPenault"/>
    <addaction name="actionPenalty_defesa"/>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuCalibrar"/>
   <addaction name="menuComandos"/>
   <addaction name="menuAjuda"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCarregar">
   <property name="text">
    <string>Carregar</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
  <action name="actionSalvar_como">
   <property name="text">
    <string>Salvar como...</string>
   </property>
  </action>
  <action name="actionFechar">
   <property name="text">
    <string>Fechar</string>
   </property>
  </action>
  <action name="actionNovo">
   <property name="text">
    <string>Novo</string>
   </property>
  </action>
  <action name="actionCores">
   <property name="text">
    <string>Cores</string>
   </property>
  </action>
  <action name="actionBordas_da_arena">
   <property name="text">
    <string>Bordas da arena</string>
   </property>
  </action>
  <action name="actionDistancias">
   <property name="text">
    <string>Dist√¢ncias</string>
   </property>
  </action>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre...</string>
   </property>
  </action>
  <action name="actionIniciar">
   <property name="text">
    <string>Iniciar</string>
   </property>
   <property name="shortcut">
    <string>I</string>
   </property>
  </action>
  <action name="actionParar">
   <property name="text">
    <string>Parar</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionPenault">
   <property name="text">
    <string>Penalty ataque</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="actionTIme_Amarelo">
   <property name="text">
    <string>Time Amarelo (Selecionado)</string>
   </property>
  </action>
  <action name="actionTime_Azul">
   <property name="text">
    <string>Time Azul</string>
   </property>
  </action>
  <action name="actionPenalty_defesa">
   <property name="text">
    <string>Penalty defesa</string>
   </property>
   <property name="shortcut">
    <string>D</string>
   </property>
  </action>
  <action name="actionFlip">
   <property name="text">
    <string>Flip (ON)</string>
   </property>
  </action>
  <action name="actionTrocar_Atacante">
   <property name="text">
    <string>Trocar Atacante</string>
   </property>
  </action>
  <action name="actionTroca_Goleiro_Defesa">
   <property name="text">
    <string>Troca Goleiro Defesa</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
